### PR TITLE
geph2 v0.20.2 (new formula)

### PR DESCRIPTION
**Pull request by uetcis**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/geph2.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Geph2 < Formula
+  desc "Modular Internet censorship circumvention system"
+  homepage "https://geph.io"
+  url "https://github.com/geph-official/geph2/archive/v0.20.2.tar.gz"
+  sha256 "fe5bef5ed04ab6f2e254e7e142ce93d57f5f1c157651db2c29aaf3cf13abf647"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```
Comment (SMillerDev) [inline Formula/geph2.rb:10]: Can this build without gopath?

File: Formula/geph2.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    bin_path = buildpath/"src/github.com/geph-official/geph2"
+    bin_path.install Dir["*"]
+    cd bin_path/"cmd/geph-client" do
+      ENV["GOOS"] = "darwin"
+      ENV["GOARCH"] = "amd64"
```
Comment (SMillerDev) [inline Formula/geph2.rb:15]: Is this required for the build?

Comment (CaptainYukinoshitaHachiman) [reply Formula/geph2.rb]: Not actually, it seems only to be necessary on the developer's CI machine which was running Linux and I just copied them without thinking. I've removed them now & Sorry for the inconvenience.

**Comment by CaptainYukinoshitaHachiman** [comment]
Thank your for reviewing and I have now removed the unnecessary code.

**Comment by chenrui333** [comment]
Squashed the commits and rebased to the master.

**Comment by chenrui333** [comment]
@CaptainYukinoshitaHachiman Thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**